2226/2/versus.rb

```diff
@@ -0,0 +1,197 @@
+require 'terminal-table'
```

Comment:
Global comment: isn't it easier to go through all files one time and collect all data u need?